src/mono/System.Private.CoreLib/src/System/RuntimeTypeHandle.cs

```diff
@@ -287,7 +287,7 @@ internal static bool IsTypeDefinition(RuntimeType type)
         private static extern RuntimeType internal_from_name(string name, ref StackCrawlMark stackMark, Assembly? callerAssembly, bool throwOnError, bool ignoreCase, bool reflectionOnly);
```

Comment:
I think this PR is incomplete as-is, and we should fix this icall up as well. Reflection-only loads should all be excised from the native side, so these bits in managed are the only remaining work and I'd like to see it all done together.

Reply:
Fixed! Thanks for the suggestion.